src/main/resources/META-INF/rewrite/java-version-17.yml

```diff
@@ -47,3 +49,41 @@ recipeList:
       version: 17
   - org.openrewrite.java.migrate.maven.UseMavenCompilerPluginReleaseConfiguration:
       releaseVersion: 17
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.DeprecatedJavaxSecurityCert
+displayName: Use javax.security.cert
+description: The javax.security.cert package has been deprecated for removal
+tags:
+  - java17
+recipeList:
+  - org.openrewrite.java.ChangeType:
+      oldFullyQualifiedTypeName: javax.security.cert
+      newFullyQualifiedTypeName: java.security.cert
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.RemovedLegacySunJSSEProviderName
+displayName: Use "SunJSSE"
+description: The com.sun.net.ssl.internal.ssl.Provider provider name was removed
+tags:
+  - java17
+recipeList:
+  - org.openrewrite.java.ChangeStringLiteral:
+      oldFullyQualifiedTypeName: com\.sun\.net\.ssl\.internal\.ssl\.Provider
+      newFullyQualifiedTypeName: SunJSSE
```

Comment:
We can pull `ChangeStringLiteral` here first, as it does not appear to be used in rewrite-liberty right now. We might want to rename that recipe to something that better reflects it's use; I'm thinking `ReplaceFirstStringLiteralMatch` (or similar), and reconsider if the arguments should really support a regular expression as input, should maybe use `Matcher.matches()` over `Matcher.find()`, and whether we indeed only want to replace the first occurrence. Things might be simpler if we start with just replacing string literals as strings anywhere in another String literal.

Reply:
Yeah, it looks like the `rewrite-liberty` repo is no longer using `ChangeStringLiteral`. I can bring it over here with a new name. The regex capabilities would be necessary for a recipe we will write sometime next year. I can remove the capability for now until it is necessary.

Reply:
I'm actually not sure `ReplaceFirstStringLiteralMatch` is a better fit. For our use case the match would need to be all or nothing. For that reason I also think leaving it as a regular expression would be best to make it more flexible for other use cases. The recipe would need a way to cover cases such as the one defined here https://github.com/openrewrite/rewrite-liberty/blob/main/src/test/java/org/openrewrite/java/liberty/InvalidInitialContextTest.java. That is a rule we plan on adding.

Reply:
I've gone and renamed the recipe in 9f02edb to align with others in other modules, with the goal being to have a uniform experience across recipes in terms of naming and arguments. hope you agree!